scripts/ontology_to_ontology_mapping_tool/ontology_to_ontology_mapping_tool.py

```diff
@@ -121,21 +121,29 @@ def validate_arguments(source_code, target_code):
         print("Please specify different ontologies to map on.\n")
         sys.exit(1)
 
+    #4. CHeck if the user input file has the source code column.
+    source_file.columns = map(str.upper, source_file.columns)
+    if source_code not in source_file.columns:
+        print("\nThe input file does not contain any ontology code columns or the headers do not match to accepted values. Please check. \n\nThe list of acceptable ontology headers are:")
```

Comment:
sorry .. one more change request ... this massage should be "does not contain the requested source ontology code column"